war/src/main/less/abstracts/colors.less

```diff
@@ -1,84 +1,150 @@
+  --warning: #ea6b19;
+  --focus: #3fb3f7;
+
+  // State colors
+  --primary-hover: #0b6aa2;
+  --primary-active: #095683;
+  --danger-hover: #eb383b;
+  --danger-active: #b50003;
+
+  // Background colors
+  --background: var(--white);
+
+  // Page header colors
+  --logo-bg: var(--white);
+  --brand-link-color: var(--secondary);
+  --header-link-color: var(--white);
+  --header-link-color-active: #f5f5f5;
+  --header-link-outline: var(--focus);
+  --header-search-border: var(--white);
+  --search-input-color: var(--brand-link-color);
+  --search-background: var(--white);
+  --search-box-completion-bg: var(--primary);
+  --search-box-shadow: 0 1px 7px 0 rgba(0, 0, 0, 0.3);
+
+  // Header classic colors
+  --header-bg-classic: #000;
+  --header-link-bg-classic-hover: #404040;
+  --header-link-bg-classic-active: #404040;
+
+  // Header new UI colors
+  --header-bg-v2: var(--primary);
+  --brand-link-color-hover-v2: var(--primary-hover);
+  --brand-link-color-active-v2: var(--primary-active);
+  --header-link-bg-hover-v2: var(--primary-hover);
+  --header-link-bg-active-v2: var(--primary-active);
+
+  // Header dimensions
+  --header-item-border-radius: 4px;
+
+  // Breadcrumbs and footer
+  --breadcrumbs-bg: #f8f8f8;
+  --breadcrumbs-border: #eaeff2;
+  --breadcrumbs-text-color: #4d545d;
+
+  // Monitor / bell-alert new colors
+  --monitor-bg-v2: #f8d7da;
+  --monitor-color-v2: #721c24;
+
+  // Alert banners
+  // Default
+  --alert-default-icon-color: #2196f3;
+  --alert-default-bg-color: #d1ecf1;
+  --alert-default-border-color: #bee5eb;
+  --alert-default-color: #0c5464;
+
+  // Success
+  --alert-success-icon-color: #4caf50;
+  --alert-success-bg-color: #d4edda;
+  --alert-success-border-color: #c3e6cb;
+  --alert-success-color: var(--success);
+
+  // Warning
+  --alert-warn-icon-color: #ff9800;
+  --alert-warn-bg-color: #fff3cd;
+  --alert-warn-border-color: #ffeeba;
+  --alert-warn-color: #856404;
+
+  // Error
+  --alert-err-icon-color: #f44336;
+  --alert-err-bg-color: #f8d7da;
+  --alert-err-border-color: #f5c6cb;
+  --alert-err-color: #721c24;
+
+  // Typography
+  --text-color: var(--black);
+  --text-color-primary: var(--primary);
+  --text-color-primary-accent: var(--primary-active);
+  --text-color-grey-light: #747c82;
+  --text-inactive: var(--text-color-grey-light);
+
+  // Buttons
+  --btn-primary-bg: var(--primary-hover);
+  --btn-primary-bg-hover: var(--primary);
+  --btn-primary-bg-active: var(--primary-active);
+  --btn-secondary-color: var(--secondary);
+  --btn-secondary-bg: var(--btn-text-color);
+  --btn-secondary-border: #9ba7af;
+  --btn-text-color: var(--white);
+
+  // Table
+  --even-row-color: #fbfbfb;
+  --bigtable-header-color: #f0f0f0;
+  --project-status-border-bottom-color: #ddd;
+  --table-parameters-background-color--hover: #f0f0f0;
+  --table-stripped-background-color--hover: #e8e8e8;
+
+  // Link
+  --link-color: #204a87;
+  --link-visited-color: #5c3566;
+
+  // Pane
+  --pane-header-color: #eee;
```

Comment:
Does this apply to all panes? For example, in a future PR I was going to change it just for side-panel widgets, dunno if this would affect it.

Reply:
it's applied to `.pane-header, .pane-footer`